indra/llui/llemojihelper.cpp

```diff
@@ -155,9 +155,12 @@ void LLEmojiHelper::setHostCtrl(LLUICtrl* hostctrl_p)
 		mHostHandle.markDead();
 		mEmojiCommitCb = {};
 
-		if (!mHelperHandle.isDead())
+		if (LLFloater* floater = mHelperHandle.get())
 		{
-			mHelperHandle.get()->closeFloater();
```

Comment:
Implementing isClosing for this feels excessive. What does this solves?

Can't you do something like this?

```
LLFloater* helper = mHelperHandle.get();
mHelperHandle.markDead()
helper->closeFloater();
```